### PR TITLE
Added code to allow users to disable HPC user creation

### PR DESCRIPTION
**Pull request by bollig**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pcluster/pcluster_stack.py [hunk 1]
```diff
@@ -192,11 +192,23 @@ def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
...
+                                   password=cdk.SecretValue.cfn_parameter(password).to_string()
+                               )
+                           )
+
+        create_user = cdk.CfnParameter(self, "CreateUser", default="false", type="String", allowed_values=['true','false']).value_as_string
+        user.cfn_options.condition = cdk.CfnCondition(self, "UserCondition", expression=cdk.Fn.condition_equals(create_user, "true"))
+
+
+        output_user_loginurl = cdk.CfnOutput(self, 'UserLoginUrl', value="".join(["https://", self.account,".signin.aws.amazon.com/console"]))
+        output_user_loginurl.condition = cdk.CfnCondition(self, "OutputUserLoginUrlCondition", expression=cdk.Fn.condition_equals(create_user, "true"))
```
Comment (sean-smith) [inline pcluster/pcluster_stack.py]: Can we re-use the same condition created above for the user?

File: pcluster/pcluster_stack.py [hunk 2]
```diff
@@ -192,11 +192,23 @@ def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
...
+        output_user = cdk.CfnOutput(self, 'UserName', value=user.ref, condition=cdk.CfnCondition(self, "OutputUserCondition", expression=cdk.Fn.condition_equals(create_user, "true")))
```
Comment (sean-smith) [inline pcluster/pcluster_stack.py]: same here

File: pcluster/pcluster_stack.py (hunk 1 above)
Comment (bollig) [reply pcluster/pcluster_stack.py]: Good point. Simplified.

File: pcluster/pcluster_stack.py (hunk 2 above)
Comment (bollig) [reply pcluster/pcluster_stack.py]: Simplified.

**Review by sean-smith — APPROVED** [review]
LGTM!

**Pull request merged**